ci-operator/config/openshift/jenkins-client-plugin/openshift-jenkins-client-master.yaml

```diff
@@ -0,0 +1,44 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
```

Comment:
i don't think you want promotion on this, right?

Reply:
(you're building a dummy jenkins image w/ a patched plugin for purposes of running the e2e, you don't want that to be promoted for general consumption).

Reply:
yeah I read the doc on that in ci-operator but still wan't entirely sure of the implication .. `configures how images that are built from the repository under test are promoted for use in tests for other repositories`  ... at the time it was unclear to me how far things went to enable "other repositories"

I'll remove